engine/build.gradle

```
@@ -150,10 +150,12 @@ dependencies {
     api group: 'org.terasology', name: 'gestalt-module', version: '5.1.5'
     api group: 'org.terasology', name: 'gestalt-util', version: '5.1.5'
     api group: 'org.terasology', name: 'gestalt-asset-core', version: '5.1.5'
-    api group: 'org.terasology', name: 'TeraMath', version: '1.4.0'
+    api group: 'org.terasology', name: 'TeraMath', version: '1.5.1-SNAPSHOT'
     api group: 'org.terasology.bullet', name: 'tera-bullet', version: '1.3.1'
     api group: 'org.terasology', name: 'splash-screen', version: '1.0.2'
     api group: 'org.terasology.jnlua', name: 'JNLua', version: '0.1.0-SNAPSHOT'
+    api group: 'org.terasology.nui', name: 'nui', version: '1.0.0'
+    api group: 'org.terasology.nui', name: 'nui-reflect', version: '1.0.0'
```

For testing the changes in https://github.com/MovingBlocks/TeraNUI/pull/12

```suggestion
    api group: 'org.terasology.nui', name: 'nui', version: '1.1.0-SNAPSHOT'
    api group: 'org.terasology.nui', name: 'nui-reflect', version: '1.1.0-SNAPSHOT'
```